backend/alert/management/commands/alertstats.py

```diff
@@ -15,7 +15,7 @@
 class Command(BaseCommand):
     help = (
         "Get statistics on PCA, and optionally send to Slack (for analytics use only; do "
-        "not confuse this script with the recomputestats command, which actually updates "
+        "not confuse this script with the recompute_soft_state command, which actually updates "
```

Comment:
It can also be run manually though - any many internally critical cronjobs like iscimport don't have that convention so I would say we should either do it for all such scripts or none